keyboards/squiggle/keymaps/rick-complicated/keymap.c

```diff
@@ -0,0 +1,197 @@
+    [QUOT_Q]        = COMBO(quot_q_combo,       KC_ENT),
+    [Q_J]           = COMBO(q_j_combo,          LCTL(KC_W)),
+    [J_K]           = COMBO(j_k_combo,          KC_DELT),
+    [Q_K]           = COMBO(q_k_combo,          KC_TAB),
+    [K_X]           = COMBO(k_x_combo,          KC_TILD),
+
+    // right hand combinations.
+    [L_R]           = COMBO(l_r_combo,          KC_BSPC),
+    [R_C]           = COMBO(r_c_combo,          KC_SLSH),
+    [C_G]           = COMBO(c_g_combo,          KC_MINS),
+    [R_G]           = COMBO(r_g_combo,          NXTTAB),
+    [G_F]           = COMBO(g_f_combo,          KC_BSLS),
+    [V_W]           = COMBO(v_w_combo,          KC_APP),
+    [W_M]           = COMBO(w_m_combo,          KC_DELT),
+    [V_M]           = COMBO(v_m_combo,          LSFT(KC_TAB)),
+    [M_B]           = COMBO(m_b_combo,          KC_GRV),
+
+    // both hand combinations.
+    [J_W]           = COMBO(j_w_combo,          KC_ENT),
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+[_BASE] = LAYOUT_complicated(
+    KC_SCLN,KC_COMM,KC_DOT, KC_P,   KC_Y,       KC_F,   KC_G,   KC_C,   KC_R,   KC_L,
+    SH_A,   CT_O,   AL_E,   KC_U,   GU_I,       GU_D,   KC_H,   AL_T,   CT_N,   SH_S,
+    KC_QUOT,KC_Q,   KC_J,   KC_K,   KC_X,       KC_B,   KC_M,   KC_W,   KC_V,   KC_Z,
+                            LW_BSPC,KC_LEAD,    ALT_ENT,RS_SPC,
+                            LW_BSPC,SFT_ESC,    ALT_ENT,RS_SPC
+),
+
+[_RAISE] = LAYOUT_complicated(
+    KC_EXLM,KC_AT,  KC_UP,  KC_LCBR,KC_RCBR,    KC_BSLS,KC_7,   KC_8,   KC_9,   KC_ASTR ,
+    KC_HASH,KC_LEFT,KC_DOWN,KC_RGHT,KC_DLR,     KC_EQL, KC_4,   KC_5,   KC_6,   KC_0 ,
+    KC_LBRC,KC_RBRC,KC_LPRN,KC_RPRN,KC_AMPR,    KC_GRV, KC_1,   KC_2,   KC_3,   KC_PLUS ,
+                            ADDDD  ,_______,    _______,_______,
+                            ADDDD  ,_______,    _______,_______
+),
+[_LOWER] = LAYOUT_complicated(
+    _______,_______,_______,KC_F1,  KC_F2,      KC_F3,  KC_F4,  _______,_______,_______ ,
+    KC_LSFT,KC_TAB, KC_PGUP,KC_F5,  KC_F6,      KC_F7,  KC_F8,  KC_HOME,KC_LALT,KC_ENT  ,
+    KC_CLCK,KC_SLCK,KC_PGDN,KC_F9,  KC_F10,     KC_F11, KC_F12, KC_END, KC_INS, _______ ,
+                            _______,_______,    _______,ADDDD,
+                            _______,_______,    _______,ADDDD
+),
+[_ADJUST] = LAYOUT_complicated(
+    _______,EXPLR,  KC_UP,  PRVTAB, PRVWIN,     NXTWIN, NXTTAB, _______,_______,LCKGUI,
+    TSKMGR, KC_LEFT,KC_DOWN,KC_RGHT,UPTAB,      DNTAB,  KC_ENT, KC_LGUI,_______,CALDL,
+    _______,CLSGUI, _______,CONPST, RESET,      _______,_______,_______,_______,_______,
+                            _______,_______,    _______,_______,
+                            _______,_______,    _______,_______
+),
+};
+
+LEADER_EXTERNS();
+
+void matrix_scan_user() {
+  LEADER_DICTIONARY() {
+    leading = false;
+    leader_end();
+
+    SEQ_ONE_KEY(KC_U) {
+      SEND_STRING(":luvu:");
+      register_code(KC_ENT);
+      unregister_code(KC_ENT);
```

Comment:
```suggestion
      SEND_STRING(":luvu:\n");
```

Reply:
the intention of that string is sending a single emoji in discord without adding a newline.

Reply:
Yes, and this does the same thing - `\n` is mapped to Enter, it won't make a new line in the text box.

Reply:
Oh, I didn't know that. thanks. please wait 10 minutes.